src/applications/vaos/appointment-list/redux/selectors.js

```diff
@@ -472,3 +477,104 @@ export function getPastAppointmentListInfo(state) {
     facilityData: state.appointments.facilityData,
   };
 }
+
+export function selectCalendarData({
+  videoKind,
+  isHome,
+  isAtlas,
+  facility,
+  appointment,
+}) {
+  const isVideo = appointment.vaos.isVideo;
+  const isCommunityCare = appointment.vaos.isCommunityCare;
+  const isInPersonVAAppointment = !isVideo && !isCommunityCare;
+
+  if (isInPersonVAAppointment) {
+    let location = '';
+    if (facility) location = formatFacilityAddress(facility);
+
+    return {
+      summary: `Appointment at ${facility?.name}`,
+      providerName: '',
+      location,
+      text: `You have a health care appointment at ${facility?.name}`,
+      additionalText: [
+        'Sign in to VA.gov to get details about this appointment',
+      ],
+    };
+  } else if (isCommunityCare) {
+    const { providerName, practiceName } =
+      appointment.communityCareProvider || {};
+
+    return {
+      summary: `Appointment at ${providerName || practiceName}`,
+      providerName: `${providerName || practiceName}`,
+      location: `${formatFacilityAddress(appointment.communityCareProvider)}`,
+      text:
+        'You have a health care appointment with a community care provider. Please don’t go to your local VA health facility.',
+      phone: `${formatFacilityPhone(appointment.communityCareProvider)}`,
+      additionalText: [
+        'Sign in to VA.gov to get details about this appointment',
+      ],
+    };
+  } else if (isVideo) {
+    const providerName = appointment.videoData?.providers
+      ? appointment.videoData.providers[0]?.display
+      : '';
+
+    if (isHome) {
+      return {
+        summary: 'VA Video Connect appointment',
+        text:
+          'You can join this meeting up to 30 minutes before the start time.',
+        location: 'VA Video Connect at home',
+        additionalText: ['Sign in to VA.gov to join this meeting'],
```

Comment:
Seems like this one would be important to get the right va.gov link to show up, so they can easily get to the video link. I wonder if we have to put urls in the text for that to work with an ics file?